services/docs-snapcraft-io.yaml

```diff
@@ -39,6 +39,12 @@ spec:
                 secretKeyRef:
                   name: sentry-dsn
                   key: docs-snapcraft-io
+          env:
+            - name: SEARCH_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: snapcraft-io-config
```

Comment:
It's very likely that this same API key will be used by many of our sites. Maybe all of them. Do you think we should call the secret "google-custom-search-api-key" as well? So it can be generically used. Or we could call it "shared-secrets"?

Reply:
It's a good question, I was browsing around and for sentry we have a sentry-dsn secret. Maybe we could have a google-api secret?

Reply:
yeah that sounds good to me

Reply:
I would prefer that too!